src/main/java/com/bartek/Medical/Mapper/VisitMapper.java

```diff
@@ -0,0 +1,16 @@
+package com.bartek.Medical.Mapper;
+
+import com.bartek.Medical.Model.Visit;
+import com.bartek.Medical.Model.VisitDto;
+import org.mapstruct.Mapper;
+
+import java.util.List;
+
+@Mapper(componentModel = "spring")
+public interface VisitMapper {
+    VisitDto toDto(Visit visit);
+    Visit toEntity(VisitDto visitDto);
+
+    List<VisitDto> toDtoList(List<Visit> visits);
```

Comment:
do przemyslenia jest to czy nie lepiej nazywac po prostu w liczbie mnogiej jesli operujemy na listach tj:
toDtos
toEntities